tpl/os/os.go

```diff
@@ -130,3 +130,22 @@ func (ns *Namespace) FileExists(i interface{}) (bool, error) {
 
 	return status, nil
 }
+
+// FileStat Stat returns the os.FileInfo structure describing file.
+func (ns *Namespace) FileStat(i interface{}) (_os.FileInfo, error) {
```

Comment:
I think we should name it simply `os.Stat`. I think we (at least has tried) to name the other funcs that more or less wraps the std lib variant with the same name as the original. In this case I think that makes sense.

Reply:
I see.
I have renamed the name.